contrib/gitian-descriptors/gitian-osx.yml

```diff
@@ -10,14 +10,20 @@ packages:
 - "git-core"
 - "pkg-config"
 - "autoconf"
+- "librsvg2-bin"
+- "libtiff-tools"
 - "libtool"
 - "automake"
 - "faketime"
 - "bsdmainutils"
 - "cmake"
+- "imagemagick"
 - "libcap-dev"
 - "libz-dev"
 - "libbz2-dev"
+- "python-dev"
+- "python-setuptools"
+- "fonts-tuffy"
 reference_datetime: "2015-06-01 00:00:00"
```

Comment:
@luke-jr `reference_datetime` was accidentally changed in master. Thus, this needs rebase. :(

Alternatively, it should be sufficient to move the `python*` packages up a bit or revert the `reference_datetime` commit in master.